### DISCUSSION TITLE
プロジェクトの進め方について

### DISCUSSION THREAD

**Discussion by hideojoho** [ir100/ir100#6] [General]
## ゴール
- プロジェクトの進め方について全体的な方針を決める
- 皆で案を出しあって、有望そうな案を選ぶ

## 動機
- 様々な立場にある有志の方々に貢献していただきたい

**Comment by hideojoho** [comment]
## 案１
- 言語処理100本ノックを参考に、各自お題（とその解法例）を提案する
    - 解法例を必須とするかは要議論
- お題がある程度（N=20ぐらい？）溜まってきたら、全体の構成（グループ分けや順序）を考える
- 適宜、お題を分解したり、統合したり、は相談しながら決める

**Comment by hideojoho** [comment]
## 案１（追記）
- お題は言語非依存が望ましいが、特定のライブラリを指定する場合はその限りでない

**Reply by tmanabe** [reply]
日本語のコンテンツになるかと思いますので、日本語特有の前処理に関する章があっても良いかもしれないですね。
